### PR TITLE
Create announcement modal to promote the dc book

### PR DESCRIPTION
**Pull request by arathjz**
**What:**
Create announcement modal to promote the dc book

**Why:**
Closes: https://app.asana.com/0/1168997577035609/1199187589531589/f

**How:**
- Open announcement if the cookie `announcementRead` does not exist or has already expired
- Set an expiration of 7 days to the `announcementRead` cookie
- Store cookie when the user closes the announcement
- Store modal content into variables

#### Media
https://www.loom.com/share/965763208e90411ab004c22f5e2f3da1
image

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ module.exports = {
         '40': '10rem'
       },
       screens: {
-        xxl: '2048px'
+        xxl: '1536px'
```
Comment (duranmla) [inline tailwind.config.js:73]: This doesn't have any consequences across the site?
Why do we need this update for the modal?

File: src/tailwind.css [hunk 2]
```diff
@@ -12,3 +12,11 @@
   font-family: 'Libre Franklin', sans-serif;
   color: #1c1c1c;
 }
+
+@media screen and (max-width: 640px) {
```
Comment (duranmla) [inline src/tailwind.css:16]: Can we map this _(640px)_ to the actually corresponding tailwind media query?
_The idea is that tailwind config will be able to tell us all the breakpoints we are really using._

**Review by duranmla — APPROVED** [review]
Couple comments that I would live to get your feedback before merge but other than that LGTM!

File: tailwind.config.js (hunk 1 above)
Comment (arathjz) [reply tailwind.config.js:73]: - It does not have consequences at all, we could consider 1536px as a big screen. `AnnouncementModal` is the only component in the whole app using `xxl:`
- I was aiming to use the same breakpoints as tailwind https://tailwindcss.com/docs/breakpoints

File: src/tailwind.css (hunk 2 above)
Comment (arathjz) [reply src/tailwind.css:16]: Do you mean using the same breakpoint for small screens as tailwind? If so, it's the same, tailwind also uses 640px for `sm` screens

Comment (orlando) [reply src/tailwind.css:16]: I think what he means is using a tailwind helper for this media query. From the Tailwind docs https://tailwindcss.com/docs/functions-and-directives#screen

```css
// same as @media (min-width: 640px) {}
@screen sm {
 
}
```

**Pull request merged**